### PR TITLE
koordlet: improve koordlet profiling and logs

### PR DESCRIPTION
**Pull request by xigang**
### Ⅰ. Describe what this PR does

koordlet: improve koordlet profiling and logs.

### Ⅱ. Does this pull request fix one issue?

```
NONE
```

### Ⅳ. Special notes for reviews

@eahydra @jasonliu747 

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: cmd/koordlet/options/options.go [hunk 1]
```diff
@@ -18,10 +18,46 @@ package options
...
 )
 
 var (
-	ServerAddr  = flag.String("addr", ":9316", "port of koordlet server")
-	EnablePprof = flag.Bool("enable-pprof", false, "Enable pprof for controller manager.")
-	PprofAddr   = flag.String("pprof-addr", ":9317", "The address the pprof binds to.")
+	ServerAddr   = flag.String("addr", ":9316", "port of koordlet server")
+	EnablePprof  = flag.Bool("enable-pprof", false, "Enable pprof for koordlet.")
+	PprofAddr    = flag.String("pprof-addr", ":9317", "The address the pprof binds to.")
+	KubeAPIQPS   = flag.Float64("kube-api-qps", 40.0, "QPS to use while talking with kube-apiserver.")
```
Comment (saintube) [inline cmd/koordlet/options/options.go:33]: how about keeping the default QPS and Burst the same as the previous config (`20, 30`)

Comment (xigang) [reply cmd/koordlet/options/options.go]: QPS and burst have been set to default values.

File: cmd/koordlet/options/options.go [hunk 2]
```diff
@@ -18,10 +18,46 @@ package options
...
+	KubeAPIBurst = flag.Int("kube-api-burst", 60, "Burst to use while talking with kube-apiserver.")
 )
+
+func installHandlerForPProf(mux *http.ServeMux) {
```
[the remark this answers is not in the document]
Comment (saintube) [reply cmd/koordlet/options/options.go:50]: Would you mind keeping the default registration of the pprof? We may not need to register the handlers manually. The timeout time can still be set by creating a server with `http.DefaultServeMux`.

[the remark this answers is not in the document]
Comment (xigang) [reply cmd/koordlet/options/options.go]: > Would you mind keeping the default registration of the pprof? We may not need to register the handlers manually. The timeout time can still be set by creating a server with `http.DefaultServeMux`.

@saintube Custom profliing server has been removed

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Review by hormes — APPROVED** [review]

**Pull request merged**